apps/participants/views.py

```diff
@@ -456,6 +458,35 @@ def get_participant_team_details_for_challenge(request, challenge_pk):
         return Response(response_data, status=status.HTTP_404_NOT_FOUND)
 
 
+@swagger_auto_schema(
+    methods=["post"],
+    manual_parameters=[
+        openapi.Parameter(
+            name="challenge_pk",
+            in_=openapi.IN_PATH,
+            type=openapi.TYPE_NUMBER,
+            description="Challenge Primary Key",
+            required=True,
+        ),
+        openapi.Parameter(
+            name="participant_team_pk",
+            in_=openapi.IN_PATH,
+            type=openapi.TYPE_NUMBER,
+            description="Participant Team Primary Key",
+            required=True,
+        ),
+    ],
+    operation_id="remove_participant_team_from_challenge",
+    responses={
+        status.HTTP_200_OK: openapi.Response(""),
+        status.HTTP_400_BAD_REQUEST: openapi.Response(
+            "{'error': 'Error message goes here'}"
```

Comment:
@afif-fahim can you replace the error message

Reply:
@Ram81 I think swagger allows a single response schema for each response code, but this API has two response schemas for 400 response code
![image](https://user-images.githubusercontent.com/33936462/109052126-7962ec00-7705-11eb-9ccc-db3c4a71866b.png)
How should I replace the error message now?

Reply:
You can mention the `Team has not participated in the challenge` response